app/helpers/orders_helper.rb

```diff
@@ -20,4 +20,8 @@ def order_detail_quantity_class(order_detail)
       "different-quantity less-quantity"
     end
   end
+
+  def bold_pending_order(order)
```

Comment:
This logic can be moved into `class Order` in `order.rb`
Then you don't have to pass in order.
The new method could be one lined in a ternary:
```ruby
def status_based_class
  pending? ? "font-bold" : ""
end
```
Or in your way (this is what @smellsblue would prefer, he dislikes ternary):
```ruby
def status_based_class
  "font-bold" if pending?
end
```